### ISSUE TITLE
noproj - needs to work on Voyager and other instruments

### ISSUE THREAD

**Issue by ascbot** [DOI-USGS/ISIS3#1902]
---

Author Name: **Tammy Becker** (Tammy Becker)

---

Larry Soderblom needs noproj to work on Voyager data. Other instruments need to be added too. It would be easier if this was generalized.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Stuart Sides** (@scsides)
Original Date: 2014-05-16T16:01:46Z

---

noproj was originally set up for HiRISe only. Some understanding of each camera will be necessary to configure and test it for each.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Tammy Becker** (Tammy Becker)
Original Date: 2014-06-09T18:09:18Z

---

For Voyager only, < 50 hours accounting for testing as well.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Tammy Becker** (Tammy Becker)
Original Date: 2014-06-09T23:02:08Z

---

Unscheduled for FY13-Q3/4.  Will reschedule when noproj is generalized to support multiple instruments.
Dropped the urgency with Larry's approval.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Tammy Becker** (Tammy Becker)
Original Date: 2015-10-07T22:50:08Z

---

Support for SocetSet DEM work with Voyager images is currently waiting on this.

**Comment by ascbot** [comment]
I am a bot that cleans up old issues that do not have activity.

Happy Birthday to this issue! :birthday:

Unfortunately, this issue has not received much attention in the last 12 months. Therefore, I am going to close it. Please feel free to reopen this issue or open a new issue sometime in the future. If this issue is a bug report, please check that the issue still exists in our newest version before reopening.
